kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/index.vue

```diff
@@ -260,8 +251,7 @@
     methods: {
       // @public
       setData(data) {
-        const { exam, difficultQuestions } = data;
-        this.quiz = exam;
+        const { difficultQuestions } = data;
```

Comment:
exam is removed here because we have a computed property which gets the data from `examMap` for us